github_server/action.yml

```diff
@@ -0,0 +1,126 @@
+name: "ALKiln/github_server: Isolated docassemble tests"
+description: "Download and start up a temporary docassemble server on GitHub's servers, in an isolated environment. Avoids docassemble reload errors and other external factors."
+
+# If we want to control the version of the docassemble image, we'd have
+# to make our own image:
+# You can build your own image by doing git clone jhpyle/docassemble  and then
+# using git checkout to select a particular version, and then using docker build
+# to build an image. https://docassemble.org/docs/docker.html#downgrading
+# If we just want to use an older version of da (as long as it works with
+# the latest image):
+# If you have an existing container you can use dainstall from the command
+# line to install docassemble.webapp==1.4.68
+
+inputs:
+  CONFIG_CONTENTS:
+    description: "The contents of the config file of the docassemble server"
+    required: false
+  MAX_SECONDS_FOR_DOCKER:
+    description: "Maximum amount of seconds to give the docassemble docker container to serve the site."
+    required: false
+    default: 600  # 10 min
+
+outputs:
+  DOCASSEMBLE_DEVELOPER_API_KEY:
+    description: "Use this inside `with:` for the ALKiln step. Temporary admin api key. It will be destroyed after the tests are done."
+    value: ${{ steps.api_key_output_step.outputs.DOCASSEMBLE_DEVELOPER_API_KEY }}
+  SERVER_URL:
+    description: "Use this inside `with:` for the ALKiln step. URL of the temporary docassemble server."
+    value: "http://localhost"
+  DA_ADMIN_EMAIL:
+    value: ${{ steps.api_key_output_step.outputs.DA_ADMIN_EMAIL }}
+  DA_ADMIN_PASSWORD:
+    value: ${{ steps.api_key_output_step.outputs.DA_ADMIN_PASSWORD }}
```

Comment:
Probably should make a README for this folder. Also probably include a README for the regular action, though I'm not sure where that would go. That would probably be in a different issue. We could also just put it in the suffolk documentation and just provide a link in the README.

Tangentially, should some kind of default github server workflow be added to the setup interview?

Reply:
Docs: Made https://github.com/SuffolkLITLab/ALKiln/issues/811
Setup: Made https://github.com/SuffolkLITLab/ALKiln/issues/812